documentation/developing-your-hyrax-based-app.md

```diff
@@ -269,3 +270,158 @@ proxy_deposit:
 
 If both options exist, whichever option is set from the Administrative Dashboard
 will take precedence.
+
+## Dockerizing a Hyrax Based App for Development
+It is now possible develop a Hyrax based app using a Docker workflow. The process utilizes **docker-compose** to build and run the services necessary for Hyrax based applications.
+
+### Switch to Postgres
+The image that you will build your app's image on uses **Postgres**. If you have just created your Hyrax-based app, for example, you'll need to update your `Gemfile`
+	- `gem "sqlite3", "~> 1.3.0"` ---> `gem "pg"`
```

Comment:
it might be worth recommending the addition of `--database = postgresql` to the `rails new` command.

Reply:
That sounds like a reasonable default to me. I can't think of a lot of situations when someone would opt to use sqlite but that's from my limited experience. I **can** see some value in adding a mysql example. But that feels like another discussion.